src/exporter/ddprof_exporter.cc

```diff
@@ -174,10 +174,8 @@ static DDRes add_single_tag(ddog_Vec_Tag &tags_exporter, std::string_view key,
   ddog_Vec_Tag_PushResult push_tag_res =
       ddog_Vec_Tag_push(&tags_exporter, to_CharSlice(key), to_CharSlice(value));
   if (push_tag_res.tag == DDOG_VEC_TAG_PUSH_RESULT_ERR) {
-    defer { ddog_Error_drop(&push_tag_res.err); };
-
-    LG_ERR("[EXPORTER] Failure generate tag (%.*s)",
-           (int)push_tag_res.err.message.len, push_tag_res.err.message.ptr);
+    log_warn_and_drop_error("[EXPORTER] Failure generate tag",
```

Comment:
Won't the error message be logged twice ?
What is the benefit of using `log_warn_and_drop_error` ?